domains/0khacviet.json

```
@@ -0,0 +1,9 @@
+{
+  "domain": "khacviet.is-a.dev",
+  "owner": {
+    "email": "viet2ca11@gmail.com"
```

username is required
```suggestion
    "username": "Khac-Viet",
    "email": "viet2ca11@gmail.com"
```